examples/overlays/add-stylized/index.html

```diff
@@ -81,24 +75,13 @@ <h4>Actions</h4>
                             <button id="btn-reset" type="button" class="btn btn-primary has-icon-right" title="Remove all the overlays of the selected BPMN element">
                                 <span class="icon icon-refresh mb-1"></span>
                             </button>
-                            <button id="btn-set-overlay" type="button" class="btn bg-secondary" title="Set custom overlay on edge or shape">
-                                <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24" width="24" height="24">
-                                    <defs>
-                                        <path d="M22.31 12.19L1.94 12.34" id="egGzqxKdQ"></path>
-                                        <path d="M8.49 12.18C8.49 10.12 10.12 8.45 12.12 8.45C14.13 8.45 15.76 10.12 15.76 12.18C15.76 14.24 14.13 15.93 12.12 15.93C10.12 15.93 8.49 14.24 8.49 12.18Z"
-                                              id="as8hBwIQ6"></path>
-                                    </defs>
-                                    <g>
-                                        <use xlink:href="#egGzqxKdQ" opacity="1" fill-opacity="0" stroke="#000000" stroke-width="2" stroke-opacity="1"></use>
-                                        <use xlink:href="#as8hBwIQ6" opacity="1" fill="#f65bc3" fill-opacity="1"></use>
-                                    </g>
-                                </svg>
+                            <button id="btn-set-overlay" type="button" class="btn btn-primary bg-warning has-icon-right" title="Set custom overlay on edge or shape">
```

Comment:
❓ as I have remove the button css style from the page, we should probably add a margin between buttons.
Previously, button had a 5px right margin. We could add a margin class to one button

Reply:
I have set a left margin to the 2nd button in f894d87